book/website/community-handbook/bookdash.md

```diff
@@ -0,0 +1,56 @@
+(ch-bookdash)=
+# _The Turing Way_ Book Dashes
+
+A book dash is a shorter version of a [Book Sprint](https://en.wikipedia.org/wiki/Book_sprint), where participants work on _The Turing Way_ book synchronously on the same day either in person or online.
```

Comment:
Feels like Book Dash should be capitalised but perhaps that's dependent on the style guide..

Reply:
I haven't yet defined it in the style guide. I think I settled for non capitalised name for the ease of writing, but I don't have a strong opinion on that.